network-api/networkapi/wagtailpages/migrations/0016_add_localization_to_subclasses.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 3.1.11 on 2021-06-10 16:55
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('wagtailcore', '0062_comment_models_and_pagesubscription'),
+        ('wagtailpages', '0015_remove_localization_from_cta_model'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='petition',
+            name='locale',
+            field=models.ForeignKey(editable=False, null=True, on_delete=django.db.models.deletion.PROTECT, related_name='+', to='wagtailcore.locale'),
```

Comment:
do we need to catch the error this will throw if someone tries to delete it while a campaign makes uses of it?

Reply:
Great question, I have no idea. @TheoChevalier thoughts on this?

Reply:
turns out: it does nothing. I tried deleting an in-use petition, and it just...worked O_o

Reply:
I’d say it’s good to have, but not blocking

Reply:
alright, let's get this in.